src/test/test-locale-util.c

```diff
@@ -119,7 +119,51 @@ TEST(dump_special_glyphs) {
         dump_glyph(SPECIAL_GLYPH_RECYCLING);
         dump_glyph(SPECIAL_GLYPH_DOWNLOAD);
         dump_glyph(SPECIAL_GLYPH_SPARKLES);
+        dump_glyph(SPECIAL_GLYPH_LOW_BATTERY);
         dump_glyph(SPECIAL_GLYPH_WARNING_SIGN);
 }
 
+#define dump_glyph_force_utf(x) log_info(STRINGIFY(x) ": %s", special_glyph(x, /* force_utf= */ true))
```

Comment:
leftover?

Reply:
> leftover?

I thought to test the "false" and "true" params?

Reply:
actually, this makes me think - instead of changing everything, you can rename `special_glyph` to `special_glyph_full` and add the new parameter, and then add a `static inline special_glyph` that has the old signature (so the existing code works and you can remove all the changes), and also another `static inline special_glyph_force_utf`

you can just use git reset to uncommiti the changes

Reply:
> actually, this makes me think - instead of changing everything, you can rename `special_glyph` to `special_glyph_full` and add the new parameter, and then add a `static inline special_glyph` that has the old signature (so the existing code works and you can remove all the changes), and also another `static inline special_glyph_force_utf`
> 
> you can just use git reset to uncommiti the changes

Okay...this hasn't sunk fully yet, perhaps cos my eyes are trying to clock out for the day.

So we have 

```const char * special_glyph_full(Specialglyph code, bool force_utf) _const_```

Then 

``` static const char * special_glyph(Specialglyph code) _const_ { return special_glyph_full(code, false)}```
And 

``` static const char * special_glyph_force_utf(Specialglyph code) _const_ { return special_glyph_full(code, true)}```

That's super genius!!!
And it's my alert self that's saying that, not someone dozing.